src/modules/Dashboard/components/ActivityGrid/ActivityGrid.types.ts

```diff
@@ -4,8 +4,12 @@ import { Roles } from 'shared/consts';
 import { DatavizActivity } from 'api';
 import { Order } from 'shared/types';
 
+import { TakeNowModalProps } from '../TakeNowModal/TakeNowModal.types';
+import { FeatureFlags } from '../../../../shared/types/featureFlags';
```

Comment:
I really need to fix this auto-import situation 😅

Reply:
If you know of a trick to make VS Code always write auto-import lines as desired – I'd love to hear it! I kinda think it would require writing a custom extension though…

Reply:
These particular ones were done by WebStorm. I'm using both because the git experience in WebStorm is just too good 🙈